app/src/main/res/values-it/strings.xml

```diff
@@ -14,16 +14,16 @@
     <string name="share_text_plant">Controlla la pianta %s nell\'app Sunflower</string>
 
     <!-- String plurals and their related prefix / suffix strings -->
-    <string name="watering_needs_prefix" translation_description="Prefix for the text describing how often a plant should be watered. Used in combination with watering_needs_suffix.">Watering needs:</string>
+    <string name="watering_needs_prefix" translation_description="Prefix for the text describing how often a plant should be watered. Used in combination with watering_needs_suffix.">Esigenze di irrigazione:</string>
     <plurals name="watering_needs_suffix">
-        <item quantity="one" translation_description="Suffix for the text describing how often a plant should be watered, singular. Used in combination with watering_needs_prefix: 'Watering needs: every day'.">every day</item>
-        <item quantity="other" translation_description="Suffix for the text describing how often a plant should be watered, plural. Used in combination with watering_needs_prefix: 'Watering needs: every 8 days'.">every %d days</item>
+        <item quantity="one" translation_description="Suffix for the text describing how often a plant should be watered, singular. Used in combination with watering_needs_prefix: 'Watering needs: every day'.">ogni giorno</item>
```

Comment:
I dont think `translation_description` is really needed, would you mind to remove it?

Reply:
Thanks, I've removed `translation_description` from the italian strings.xml file.

Reply:
thx